src/DynamoCoreWpf/ViewModels/Core/NoteViewModel.cs

```diff
@@ -105,6 +113,7 @@ public NoteViewModel(WorkspaceViewModel workspaceViewModel, NoteModel model)
             model.PropertyChanged += note_PropertyChanged;
             DynamoSelection.Instance.Selection.CollectionChanged += SelectionOnCollectionChanged;
             ZIndex = ++StaticZIndex; // places the note on top of all nodes/notes
+            IsOnEditMode = false;
```

Comment:
Do we need to set this to false in the constructor? a bool property should default to false